1-org/README.md

```diff
@@ -72,3 +72,30 @@ When using Cloud Build or Jenkins as your CI/CD tool each environment correspond
 1. Run `./tf-wrapper.sh apply production`
 
 If you received any errors or made any changes to the Terraform config or `terraform.tfvars` you must re-run `./tf-wrapper.sh plan production` before run `./tf-wrapper.sh apply production`
+
+### Optional Variables
+
+Some variables used to deploy the step have default values. Check if you want to change those values before doing the deploy:
+
+- **skip_gcloud_download:** By default this option is `true`. The default value assumes gcloud is already available outside the module, if you change to false it will download gcloud.
```

Comment:
From a maintainability perspective, I dont think handwriting these are really sustainable. I feel like we can either
- highlight important ones
- use a custom [terraform-docs](https://github.com/terraform-docs/terraform-docs/blob/master/docs/CONFIG_FILE.md) config to autogen some version of this.

Reply:
+1 we should definitely auto-generate these docs with the same convention we are using elsewhere. There should be a better way to highlight optional variables